sources/packages/backend/apps/api/src/route-controllers/institution-locations/models/application.dto.ts

```diff
@@ -44,6 +47,15 @@ export class ActiveApplicationSummaryAPIOutDTO {
   applicationScholasticStandingStatus: ApplicationScholasticStandingStatus;
 }
 
+/**
+ * Query string options available for active applications summary.
+ */
+export class ActiveApplicationSummaryAPIQueryStringDTO extends ApplicationStatusPaginationOptionsAPIInDTO {
+  @IsBoolean()
+  @ToBoolean()
```

Comment:
Trying to understand why we are using `@ToBoolean` here. 
Params like page and pageLimit are casted as number in the DTO.

Reply:
They are nicely cast when DTOs ate from the body payload where a boolean is actually received as a boolean.
```ts
{
    // This will be cast to a boolean.
    myBolleanProperty: true
}
```
When the values come from the query string, all of them are strings, which will make any string different than the "empty string" `true`.

Reply:
does this part of the wava scan suggestion?

Reply:
> does this part of the wava scan suggestion?

Just replying here as history since we had that clarify over a call, yes, this was needed otherwise the endpoint would fail because the `archive` was not defined in the DTO.

Reply:
Let me know if I am missing something. I get that values coming from query params are string and was trying to understand why we are not explicitly casting params like pageLimit and page which are numbers and only doing that for boolean field?

Reply:
> 

Maybe it is better to look on the below example:
![image](https://github.com/bcgov/SIMS/assets/61259237/237ab9b8-eba4-4d4c-8c3e-5c4c29bef276)